Sources/NIOHTTP1/HTTPDecoder.swift

```diff
@@ -17,10 +17,9 @@ import CNIOHTTPParser
 
 private struct HTTPParserState {
     var dataAwaitingState: DataAwaitingState = .messageBegin
-    var currentHeaders: HTTPHeaders?
+    var currentHTTPHeaders: [HTTPHeaderIndex] = []
```

Comment:
nit: i would call this `currentHeaderIndexes`. `HTTP` is redundant and this doesn't actually contain headers.

Reply:
@tanner0101 I used `currentHeaderIndices`... thanks for the suggestion :)